# Always camel case every thing (`jlc/always-camel-case`)

<!-- end auto-generated rule header -->

## Rule Details

This rule aims to enforce camal case in naming things.

Examples of **incorrect** code for this rule:

```js
let a_b = 1;
```

Examples of **correct** code for this rule:

```js
let aB = 1;
```
