# Kebab case your file name and directory name (`jlc/kebab-case`)

<!-- end auto-generated rule header -->

If you naming filename like this, it's bad case:

```js
renderDom.js / UserManagement.html;
```

That is good:

```js
render - dom.js / signup.css / index.html / company - logo.png;
```
