# Prefer template literal (`jlc/prefer-template-literal`)

ðŸ’¼ This rule is enabled in the âœ… `recommended` config.

<!-- end auto-generated rule header -->

That is bad:

```js
const name = "tom";
const message = "Hello" + name;
```

That is good:

```js
const name = "tom";
const message = `Hello ${name}`;
```
